src/components/RangeInput.vue

```diff
@@ -1,15 +1,20 @@
 <template>
 <div v-if="state" :class="suit()">
-  <form :class="suit('form')" @submit.prevent="refine()">
+  <slot
+    :currentRefinements="this.state.start"
+    :refine="this.refine"
+  >
+  <form :class="suit('form')" @submit.prevent="refine(minInput, maxInput)">
     <label :class="suit('label')">
-      <input :class="[suit('input'), suit('input', 'min')]" type="number" ref="minInput" :min="this.min" :placeholder="this.min"/>
+      <input :class="[suit('input'), suit('input', 'min')]" type="number" :min="this.min"  :placeholder="this.min" :value="this.state.start && this.state.start[0]" @change="minInput = $event.currentTarget.value"/>
     </label>
     <span :class="suit('separator')">to</span>
     <label :class="suit('label')">
-      <input :class="[suit('input'), suit('input', 'max')]" type="number" ref="maxInput" :max="this.max" :placeholder="this.max"/>
+      <input :class="[suit('input'), suit('input', 'max')]" type="number" :max="this.max"  :placeholder="this.max" :value="this.state.start && this.state.start[1]" @change="maxInput = $event.currentTarget.value"/>
     </label>
     <button :class="suit('submit')" type="submit">Go</button>
```

Comment:
We can use a slot to allow the customisation of the label.

Reply:
Do you want an empty slot for strictly adding a label or do you want a slot on `<label>`?

Reply:
Rather than an hard coded value for the text we can create a slot under the button to allow the user to customise it.

```html
<button :class="suit('submit')" type="submit">
  <slot name="nameOfTheSlots">Go</slot>
</button>
```